### PR TITLE
Transpiled UTs

### PR DESCRIPTION
**Pull request by sbcgua**

### CODE REVIEW

**Comment by sbcgua** [comment]
@larshp Can I ask you to help me with this? I'm trying to transpile the code but no ajson class is actually included and so no UTs run consequently.

See the script in [workflows/test.yml](https://github.com/sbcgua/ajson/pull/167/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88)

P.S. I intentionally don't commit the package.json, but it is created during the installation

File: transpile_for_testing.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "input_folder": "{src}",
+  "input_filter": [
+    "src/core/*",
+    "src/libs/*"
```
Comment (larshp) [inline transpile_for_testing.json:5]: try this,

```suggestion
  "input_folder": "src",
  "input_filter": [
    "core/*",
    "libs/*"
```

**Comment by sbcgua** [comment]
Nice ! It works :)

File: README.md [hunk 2]
```diff
@@ -1038,6 +1038,7 @@ Behavior options like `read_only` or `keep_item_order` are accessible via `opts(
 ## References
 
 - Forked from [here](https://github.com/abaplint/abaplint-abap-backend) originally, at early stages
+- The package is unit tested dynamically in Github actions thanks to the [abap transpiler](https://github.com/abaplint/transpiler) by @larshp
```
Comment (sbcgua) [inline README.md:1041]: @larshp

Comment (larshp) [reply README.md:1041]: nice

**Pull request merged**